Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -94,11 +94,27 @@ def putMaskOnVTKGrid(data, grid, actorColor=None, cellData=True, deep=True):
             grid.GetPointData().AddArray(ghost)
     return mapper
 
+def handleProjectionEdgeCases(projName, data):
+    # For mercator projection, latitude values of -90 or 90
+    # transformation result in infinity values. We chose -85, 85
```

Comment:
sure, yup, I knew -:) but you are right not everyone would know. On related note, matplotlib has a threshold option as well that I would like to add at some point.